docs/examples/cooldowns.js

```diff
@@ -0,0 +1,59 @@
+/*
+  A ping pong bot, whenever you send "ping", it replies "pong".
+  However, you can only do it so quickly.
+*/
+
+// Import the discord.js module
+const Discord = require('discord.js');
+
+// Create an instance of a Discord client
+const client = new Discord.Client();
+
+// The token of your bot - https://discordapp.com/developers/applications/me
+const token = 'your bot token here';
+
+// The ready event is vital, it means that your bot will only start reacting to information
+// from Discord _after_ ready is emitted
+client.on('ready', () => {
+  console.log('I am ready!');
+});
+
+// Cooldown User Object (Empty Array)
+var cooldownUsers = [];
+
+// Cooldown Check Function
+const checkCooldown = ((userId) => {
```

Comment:
I don't really think the entire function is useless.  Some people don't really know how to create cooldowns for their commands when they are learning how to code.